src/client/dfuse/ops/readdir.c

```diff
@@ -759,6 +761,12 @@ dfuse_do_readdir(struct dfuse_info *dfuse_info, fuse_req_t req, struct dfuse_obj
 			}
 
 			if (drc) {
+				if (oh->doh_rd_nextc != NULL)
```

Comment:
minor nit, any time I have comments, I use braces on if statement and checkpatch never cared.   I personally hate the no braces for single line rule style but that is another story.

Reply:
I am with you. I will fix once i need to refresh the patch. Probably after @ashleypittman's review.

Reply:
I prefer the brevity of no braces for single-line-if statements as they're often simple but I completely agree that if there are comments involved it needs to be clear what code block they belong to.